warp_drive/utils/argument_fix.py

```diff
@@ -0,0 +1,33 @@
+import functools
+import warnings
+
+
+class Argfix:
+    """Decorator ensuring backward compatibility when an argument name is
```

Comment:
Can you clarify when we modify argument names? That's kinda dangerous...

Reply:
added comments